README.md

```diff
@@ -72,10 +72,12 @@ Here is the complete code to create this. This is all done in one Python file!
 import reflex as rx
 import openai
 
-openai.api_key = "YOUR_API_KEY"
+openai_client =  openai.OpenAI(api_key="YOUR_API_KEY")
```

Comment:
I think in the new api you don't need to pass in the key, it just reads it from the env var. (Maybe we can just put a comment telling them to make sure its set)